app/src/main/java/protect/card_locker/GroupCursorAdapter.java

```diff
@@ -43,8 +44,18 @@ public void onBindViewHolder(GroupListItemViewHolder inputHolder, Cursor inputCu
         inputHolder.mName.setText(group._id);
 
         int groupCardCount = DBHelper.getGroupCardCount(mDatabase, group._id);
-        inputHolder.mCardCount.setText(mContext.getResources().getQuantityString(R.plurals.groupCardCount, groupCardCount, groupCardCount));
+        int archivedCardCount = DBHelper.getArchivedCardsCount(mDatabase, group._id);
 
+        Resources resources = mContext.getResources();
+
+        String mCardCountText;
```

Comment:
Just a small thing, can you please use `cardCountText` instead of `mCardCountText`? I use `mVariableName` instead of `variableName` only for variables that are part of the class itself (I'm not sure why `m`, I inherited that from Loyalty Card Keychain)

Reply:
The changes requested have been made. @TheLastProject